### DISCUSSION TITLE
blog/web/what-is-yarn-berry

### DISCUSSION THREAD

**Discussion by giscus** [deokgoo/next-duck-blog#7] [General]
# blog/web/what-is-yarn-berry

yarn berry, yarn2, yarn 버전3, 자세히 파보자

https://duck-blog.vercel.app/blog/web/what-is-yarn-berry#comments-container

**Comment by Mellbrown** [comment]
감사합니다 땡큐

**Comment by Mellbrown** [comment]
다음글 언제 올라오나요?

**Reply by ch1517** [reply]
222 기다리고 있습니다!

**Reply by Mellbrown** [reply]
yarn berry berry~ berry~ yarn yarn yarn
